src/core/ipns/routing/pubsub-datastore.js

```diff
@@ -0,0 +1,171 @@
+'use strict'
+
+const ipns = require('ipns')
+const { fromB58String, toB58String } = require('multihashes')
+const PubsubDatastore = require('datastore-pubsub')
+
+const withIs = require('class-is')
+
+const errcode = require('err-code')
+const debug = require('debug')
+const log = debug('jsipfs:ipns:pubsub')
+log.error = debug('jsipfs:ipns:pubsub:error')
+
+const ipnsNS = '/ipns/'
+const ipnsNSLength = ipnsNS.length
```

Comment:
i predict these 2 lines will be used in more places maybe they can be exported from the ipns package ?